lib/meta-box-partial-page.php

```diff
@@ -113,14 +113,20 @@ function gutenberg_filter_meta_boxes( $meta_boxes ) {
 	$core_normal_meta_boxes = array(
 		'revisionsdiv',
 		'postexcerpt',
-		'postcustom',
 		'trackbacksdiv',
 		'commentstatusdiv',
 		'commentsdiv',
 		'slugdiv',
 		'authordiv',
 	);
 
+	// Whether or not to load the 'postcustom' meta box is stored as a site option
+	// so that we're not always loading its assets.
+	$enable_custom_fields = get_option( 'enable_custom_fields', false );
```

Comment:
Should this be `get_user_meta()` instead of `get_option()`? With the latter, it seems like you could have inconsistent behavior where User A could turn off the meta box for User B, and then User B turns it back on for User A, etc.

Reply:
You're totally right. Fixed in b02581c3e.